app/templates/gis-search-form.hbs

```diff
@@ -11,6 +11,7 @@
     onShow=(action "onErrorShow")
```

Comment:
This `ui-message` became unnecessary now, and should be removed with it's action handlers.